src/main/java/org/fundacionjala/pivotal/api/APIRequestManager.java

```diff
@@ -0,0 +1,71 @@
+package org.fundacionjala.pivotal.api;
+
+import com.jayway.restassured.response.Response;
+import java.util.Map;
+import static com.jayway.restassured.RestAssured.given;
+
+/**
+ * APIRequestManager.java
+ * Class that implements the methods to get, post, delete and put through Pivotal API.
+ */
+public final class APIRequestManager {
+    /**
+     * Constructor.
+     */
+    private APIRequestManager() {
+    }
+
+    /**
+     * static attribute to manage project's id through processes.
+     */
+    private static String projectId = "0";
+
+    /**
+     * Response attribute to get request.
+     */
+    private static Response response;
+
+    /**
+     * Method to post a request.
+     * @param values map.
+     * @param endpoint to request to.
+     */
+    public static void post(final Map<String, String> values, final String endpoint) {
+        APIRequestManager.response = given()
+                .spec(APIManager.getInstance().getRequestSpecification())
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)